### ISSUE TITLE
Failed to get module version from ${packageJsonPath}

### ISSUE THREAD

**Issue by Zeph33** [tyhopp/patch-files#11]
patch-file  don't find package.json for module @quasar/quasar-app-extension-qoverlay
package.json location is node_modules/@quasar/quasar-app-extension-qoverlay/package.json
```
npx patch-files@latest node_modules/@quasar/quasar-app-extension-qoverlay/src/index.js
file:///home/zeph/.npm/_npx/c636e39941a2a9a5/node_modules/patch-files/src/lib/get-module-version.js:18
      `Failed to get module version from ${packageJsonPath}`
                                           ^

ReferenceError: packageJsonPath is not defined
    at getModuleVersion (file:///home/zeph/.npm/_npx/c636e39941a2a9a5/node_modules/patch-files/src/lib/get-module-version.js:18:44)
    at async createPatches (file:///home/zeph/.npm/_npx/c636e39941a2a9a5/node_modules/patch-files/src/create-patches.js:21:21)
    at async patchFiles (file:///home/zeph/.npm/_npx/c636e39941a2a9a5/node_modules/patch-files/src/patch-files.js:6:5)

Node.js v20.17.0
```

**Comment by tyhopp** [comment]
Thanks for raising this issue, I was able to reproduce it. The reason it fails is the file path resolution logic does not handle cases where npm module names have "@" in them.

I won't have time to work on this, but if you'd like to put up a PR with tests for this case I'll carve out time to review.

**Comment by tyhopp** [comment]
The fix for this has been released in [6.0.5](https://www.npmjs.com/package/patch-files/v/6.0.5), thank you! @Zeph33 would you like to be added to a `CONTRIBUTORS.md` file?

**Comment by Zeph33** [comment]
> The fix for this has been released in [6.0.5](https://www.npmjs.com/package/patch-files/v/6.0.5), thank you! @Zeph33 would you like to be added to a `CONTRIBUTORS.md` file?

Yes, if you want
